purchase_delivery_split_date/models/purchase_order_line.py

```diff
@@ -3,11 +3,10 @@
 # Copyright 2021 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
-from itertools import groupby
```

Comment:
This change is also strange:
`from itertools import groupby` to `from odoo.tools import groupby`.
This `gropuby` isn't interchangeable.
Can you pls explain?